### PR TITLE
Milestone 1

### PR DESCRIPTION
**Pull request by Mihndim2020**
- Created two directories, bin, and lib
- Created two files; .gitkeep and main.rb
- Make main.rb executable

### CODE REVIEW

File: bin/main.rb [hunk 1]
```diff
@@ -0,0 +1 @@
+puts 'Hello World!'
```
Comment (CalyCherkaoui) [inline bin/main.rb:3]: A line with commented `#!/usr/bin/env ruby` is required here at the top of the main.rb file.

**Review by CalyCherkaoui — CHANGES_REQUESTED** [review]
## REQUIRED CHANGES ♻️ 
Hi Julius, hi Angela 😄  This is your first step to build the Tic-tac-toe game. You are doing a great job here. 👏🏼 
However, you need to make a few changes to complete this milestone 🍀 
- [x] ``bin/main.rb`` file needs to be executable. You will find here a good tutorial to follow :  [Making a Ruby Script Executable](https://commandercoriander.net/blog/2013/02/16/making-a-ruby-script-executable/)
- [x] The <b>development branch</b> need to be declared as a <b>default branch</b>
<hr>
<b>Good luck with your project guys</b> 👍🏼 🍀  
<br>
Please request another review when you finish checking the requested changes above.
<br>
PS: I gave you a little hint in the review comment in your code (bin/main.rb file). Don't forget to check it 😉 
<br>
<br>
Happy coding!

**Comment by Mihndim2020** [comment]
-File made executable
- Development branch as default

**Review by ElijahTobs — APPROVED** [review]
# Status: Approved! :tada: 

## Congratulations! @Mihndim2020 and @angelacuahutle 

 Good job implementing the change as requested by the previous TSE.

## Good luck on the next milestone.

## Happy Coding :computer: 

_Feel free to reach out to me on slack_ **`@Elijah Ayandokun`**
  **Elijah O. Ayandokun**
[Github](https://github.com/ElijahTobs) | [LinkedIn](https://www.linkedin.com/in/ayandokunelijah/) | [Twitter](https://twitter.com/ElijahTobs)

**Pull request merged**